src/framework/uicomponents/view/validators/intinputvalidator.cpp

```diff
@@ -48,7 +48,7 @@ QValidator::State IntInputValidator::validate(QString& inputStr, int& cursorPos)
 {
     QValidator::State state = Invalid;
 
-    if (inputStr.contains(QRegularExpression("^\\-?\\d{1,3}$"))) {
+    if (inputStr.contains(QRegularExpression("^\\-?\\d{1,4}$"))) {
         if (inputStr.contains(QRegularExpression("^\\-?0{2,3}"))
```

Comment:
should be `2,4`

Reply:
Why? as far as I can tell single digit DPI values are alowed?

As, I see, they apparently were in Mu3, but in Mu4 the minimum is 32

Reply:
Hey, I apologize for not addressing this comment earlier. 
The validator is specifically linked to `IncrementalPropertyControl.qml`, and there are other properties with minimum values in single digits.
When I set this validator to 1, 4, the `IntInputValidator::fixup` method inside `intinputvalidator.cpp` will automatically adjust it to the minimum value for DPI if the value entered is below 32. 
Using `2, 4` might lead to issues with properties that have minimum values in single digits, for example, the insert measure popup.

Reply:
Perhaps, We could try to define these values as variables and pass a property to QML, which can then set these variables if it's possible with QRegularExpression.

Reply:
There might be a misunderstanding here. As I understand it, the comment is about line 52, not line 51.
```suggestion
        if (inputStr.contains(QRegularExpression("^\\-?0{2,4}"))
```
Is that true @Eism?

Reply:
Also, instead of '4' we might want to use something in the direction of `std::ceil(std::log(std::max(std::abs(m_top), std::abs(m_bottom))))`, isn't that better?

Reply:
> Also, instead of '4' we might want to use something in the direction of `std::ceil(std::log(std::max(std::abs(m_top), std::abs(m_bottom))))`, isn't that better?

Thank you, Casper! That's exactly what I needed. I'll test it out and make a commit